src/main/java/org/lightcouch/internal/URIBuilder.java

```diff
@@ -72,7 +88,7 @@ public URI buildEncoded() {
 		}
 		try {
 			String q = (query.length() == 0) ? "" : "?" + query;
-			String uri = String.format("%s://%s:%s%s%s%s", new Object[]{scheme, host, port, path, pathToEncode, q});
+			String uri = String.format("%s://%s:%s%s%s%s", new Object[]{scheme, host, port, path, encodedPath, q});
```

Comment:
I know this wasn't in your change but can we use the varargs syntax here instead (remove `new Object[]{}`)?